debezium-core/src/test/java/io/debezium/transforms/outbox/EventRouterTest.java

```diff
@@ -710,7 +710,7 @@ public void canSetPartitionWithAdditionalFields() {
     public void shouldFailOnInvalidConfigurationForTopicRegex() {
         final EventRouter<SourceRecord> router = new EventRouter<>();
         final Map<String, String> config = new HashMap<>();
-        config.put(EventRouterConfigDefinition.ROUTE_TOPIC_REGEX.name(), " [[a-z]");
```

Comment:
Weird, does re2j considers this as a valid regex? 😄

Reply:
Yes, this is considered valid in re2j. I have checked it with some of the online matchers and they also return this as valid.